cou/apps/base.py

```diff
@@ -793,3 +791,28 @@ def _check_application_target(self, target: OpenStackRelease) -> None:
                 f"Application '{self.name}' already configured for release equal to or greater "
                 f"than {target}. Ignoring."
             )
+
+    def _check_mismatched_versions(self, units: Optional[list[COUUnit]]) -> None:
+        """Check that there are no mismatched versions on app units.
+
+        If units are passed that means that the application will upgrade unit by unit,
+        so there are no mismatch.
```

Comment:
I don't think it is correct to say there's no mismatch in unit-by-unit upgrade. I suggest reword this to:
```suggestion
        If units are passed, the application will upgrade in unit-by-unit fashion,
        and mismatch is not checked.
```